### ISSUE TITLE
[Question] One of the devices can't locate image targets, why?

### ISSUE THREAD

**Issue by AytoMaximo** [Unity-Technologies/arfoundation-samples#1077]
I have 5 "iPad Pro 11" devices. I've installed the same Unity game app with ARFoundation module on all of them. All of the iPads work fine and are capable of image tracking, however, one of them can't see the image target at all. 

All of the iPads were tested on the same target and in the same environment. 

Does anyone have any hypotheses as to why this might be happening?

**Comment by andyb-unity** [comment]
This sounds like a hardware failure. You could verify that the `TrackedImageManager` in your scene has a non-null subsystem: https://docs.unity3d.com/Packages/com.unity.xr.arfoundation@5.0/api/UnityEngine.XR.ARFoundation.SubsystemLifecycleManager-3.html#UnityEngine_XR_ARFoundation_SubsystemLifecycleManager_3_subsystem

If so, at that point image detection is handled by iOS directly.

**Comment by AytoMaximo** [comment]
> This sounds like a hardware failure. You could verify at runtime that the `TrackedImageManager` in your scene has a non-null subsystem: https://docs.unity3d.com/Packages/com.unity.xr.arfoundation@5.0/api/UnityEngine.XR.ARFoundation.SubsystemLifecycleManager-3.html#UnityEngine_XR_ARFoundation_SubsystemLifecycleManager_3_subsystem
> 
> If so, at that point image detection is handled by iOS directly.

Apologies for the delayed response. I initially thought it was a phantom hardware failure, but the problem recurred when more iPads were purchased. The TrackedImageManager certainly has a non-null subsystem; I've checked the logs, so I believe the problem lies somewhere in ARFoundation.

Additionally, I captured a screenshot and noticed that the camera feed is unfocused. My initial thought was to disable autofocus in the AR Camera Manager, but that didn't work.

![image](https://github.com/Unity-Technologies/arfoundation-samples/assets/14392087/cc043616-5615-472b-9a61-dfa0143b92b9)

![image](https://github.com/Unity-Technologies/arfoundation-samples/assets/14392087/629134b8-d3d1-42e8-b8c0-ee67c74feaa4)
